### PR TITLE
Create component pageTitle

### PR DESCRIPTION
**Pull request by manuugm**
Link Trello: https://trello.com/c/RNQ2uVft/15-t%C3%ADtulo-de-p%C3%A1gina-p%C3%A1ginas-de-contenido-3
Link Zeplin: https://app.zeplin.io/project/5f15bfe784a1689e60712353/screen/5f15c0228f1d6e9cee118eb9

- Create pageTitle.js file with the component structure 
- Create styles for the page title component 
- Add pageTitle data to data.js
- Fix syntax errors in _variables.scss and _typography.scss
image

### CODE REVIEW

File: src/js/components/pageTitle.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const pageTitle = data => {
+  const section = document.createElement('section');
```
Comment (omeneses) [inline src/js/components/pageTitle.js:2]: Tag names should be uppercase. Example:   'SECTION'

File: src/js/components/pageTitle.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  const eyebrow = document.createElement('p');
```
Comment (omeneses) [inline src/js/components/pageTitle.js:3]: Tag names should be uppercase.

File: src/js/components/pageTitle.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  const heading = document.createElement('h1');
```
Comment (omeneses) [inline src/js/components/pageTitle.js:4]: Tag names should be uppercase.

File: src/data/data.js [hunk 4]
```diff
@@ -15,14 +15,19 @@ const pageData = {
           href: 'Register'
         }
       },
+      {
+        component: 'page-title',
```
Comment (camigomezdev) [inline src/data/data.js]: Please use camelCase format to name the components.

File: src/scss/abstracts/_variables.scss [hunk 5]
```diff
@@ -10,5 +10,4 @@ $green-dark: #043326;
 $grey: #d8d8d8;
 $orange: #ffcc36;
 $yellow: #ffcc36;
-$white: #ffffff;
-
+$white: #fff;
```
Comment (camigomezdev) [inline src/scss/abstracts/_variables.scss:13]: please add an empty line to the end of the file

File: src/js/components/pageTitle.js (hunk 1 above)
Comment (jsebax) [reply src/js/components/pageTitle.js:2]: There is no difference between uppercase and lowercase tag names in the `createElement` method. As a way to keep consistency between the code, you can use the tag names here in lowercase.

File: src/js/components/pageTitle.js (hunk 2 above)
Comment (jsebax) [reply src/js/components/pageTitle.js:3]: There is no difference between uppercase and lowercase tag names in the `createElement` method. As a way to keep consistency between the code, you can use the tag names here in lowercase.

File: src/js/components/pageTitle.js (hunk 3 above)
Comment (jsebax) [reply src/js/components/pageTitle.js:4]: There is no difference between uppercase and lowercase tag names in the `createElement` method. As a way to keep consistency between the code, you can use the tag names here in lowercase.

File: src/scss/abstracts/_variables.scss (hunk 5 above)
Comment (manuugm) [reply src/scss/abstracts/_variables.scss:13]: Done!

File: src/data/data.js (hunk 4 above)
Comment (manuugm) [reply src/data/data.js]: Done!

**Review by jsebax — APPROVED** [review]

**Review by juan-ortizc — APPROVED** [review]

File: src/data/data.js (hunk 4 above)
Comment (omeneses) [reply src/data/data.js]: It still requires to change to camelCase.
Screen Shot 2020-08-07 at 7 47 19 PM

**Review by omeneses — APPROVED** [review]

**Pull request merged**